tests/requirements.txt

```diff
@@ -1,3 +1,4 @@
 pytest==7.4
 pytest-forked==1.6
 scipy~=1.10
+neuron
```

Comment:
no need to add `neuron` here as not all the tests require `neuron`, e.g. `tests/unit`.